bedrock/firefox/templates/firefox/firstrun/new-firstrun.html

```diff
@@ -12,6 +12,8 @@
   {% javascript 'firefox_new_firstrun' %}
 {% endblock %}
 
+{% block global_nav %}{% endblock %}
+
```

Comment:
If that is the case (using `firstrun-horizon.html` with the FxA UI from f=99), wouldn't we need to conditionally update the `entrypoint` param as well? (I'm totally guessing at that `entrypoint` param here - it's the first noticeable difference I saw between the implementations.)